.github/workflows/deploy_to_aws.yml

```diff
@@ -31,6 +31,11 @@ on:
         required: false
         type: boolean
         default: false
+      clear-opensearch:
+        description: 'Clear OpenSearch index'
```

Comment:
Let's make it plural — **indexes**. In the near future, we will have a few more indexes, so it won't just be one.

Reply:
done